.builds/freebsd.yml

```diff
@@ -1,5 +1,6 @@
 image: freebsd/latest
 packages:
+- devel/basu
```

Comment:
The package is still missing probably because I've recently moved it.
https://svnweb.freebsd.org/changeset/ports/559264